x-pack/test/functional/services/ml/api.ts

```diff
@@ -1504,9 +1504,8 @@ export function MachineLearningAPIProvider({ getService }: FtrProviderContext) {
 
     async deleteIngestPipeline(modelId: string, usePrefix: boolean = true) {
       log.debug(`Deleting ingest pipeline for trained model with id "${modelId}"`);
-      const { body, status } = await esSupertest.delete(
-        `/_ingest/pipeline/${usePrefix ? 'pipeline_' : ''}${modelId}`
-      );
+      // const { body, status } =
```

Comment:
```suggestion
```

Reply:
Fair suggestion, feel free to apply it

Reply:
non issue, can be addressed later if we want to expedite merging.

Reply:
whoops I should reload before commenting.  I'll let @Ikuni17 decide

Reply:
I think we can fix it with #167392

Reply:
Added https://github.com/elastic/kibana/pull/167392/commits/5f987f177341471e7a65a5f4172e12d1ed46090d